tests/test_generators/test_rustgen_maturin.py

```diff
@@ -0,0 +1,128 @@
+import os
+import shutil
+import subprocess
+from pathlib import Path
+
+import pytest
+from linkml_runtime.utils.introspection import package_schemaview
+
+from linkml.generators.rustgen import RustGenerator
+
+
+@pytest.mark.rustgen
+def test_rustgen_personschema_maturin(input_path, temp_dir):
+    """
+    Generate a Rust crate from personinfo.yaml and try to build
+    the Python bindings with maturin. Skips if maturin/cargo are absent.
```

Comment:
i think we shouldn't double-skip here: you already are skipping in the conftest, so if have reached this test we are explicitly intending the rustgen tests to run. so if we reach here and still don't have the appropriate dependencies, we should fail.